render.yaml

```diff
@@ -19,9 +19,17 @@ services:
       fromDatabase:
         name: issuing-treasury-db
         property: connectionString
-    - key: NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY
+    - key: NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY_US
       sync: false
-    - key: STRIPE_SECRET_KEY
+    - key: NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY_UK
+      sync: false
+    - key: NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY_EU
+      sync: false
+    - key: STRIPE_SECRET_KEY_US
+      sync: false
+    - key: STRIPE_SECRET_KEY_UK
+      sync: false
+    - key: STRIPE_SECRET_KEY_EU
```

Comment:
Thinking about the UX for a non-dev deploying to Render. Now they'll be prompted to enter 6 keys instead of 2. We should update the README and docs once dev work is done to guide that they don't necessarily have to enter all 6 keys. They can for example just choose to use only the US keys and leave the others alone. And UK devs can do the opposite. Could we make it such that our code can automatically disable UK selection if for example the UK platform key is not present? Maybe it already does that and you can point me to it?

Reply:
Great call

Reply:
added in b84e1e369ece926ceba5b6d796a81596dc009b17, (with a fallback in 79f35f1030f63290b1d44352966641758d021159 for compatibility in existing environments that haven't updated key env vars yet)

Reply:
(I swear GitHub used to automatically create links for text strings that look like commit SHAs?)